.snyk

```diff
@@ -12,4 +12,9 @@ ignore:
         reason: No upgrade available
         expires: 2024-07-18T00:00:00.000Z
         created: 2023-07-18T12:52:32.034Z
+  SNYK-JS-SNYKAZUREPIPELINESTASK-5867274:
+    - '*':
+        reason: No upgrade or patch available yet
```

Comment:
Let's reference the pending fix PR in the reason:
`reason: Pending fix - https://github.com/microsoft/azure-pipelines-task-lib/pull/961`